README.md

```diff
@@ -96,35 +97,35 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Question:** What is happening on the first couple of lines of the new file you've created?
 
-* **Your Answer:** 
+* **Your Answer:** setting the environment variables 
 
 ---
 
 - [ ] Let's store the token in LocalStorage with a key of `journal-app`.
 
 * **Question:** Why are we storing the token?
 
-* **Your Answer:**
+* **Your Answer:** so we can ask for it every time
```

Comment:
Storing it locally means we do not have to make a server request every time we refresh the page and interrupt the users flow.